src/Cody.VisualStudio.Tests/SmartApplyTests.cs

```diff
@@ -38,57 +38,66 @@ public SmartApplyTests(ITestOutputHelper output) : base(output)
         }
 
         [VsFact(Version = VsVersion.VS2022)]
-        public async Task Apply_Suggestion_Is_Modifying_Point_Document()
+        public async Task Apply_Suggestion_Is_Modifying_Dummy_Document()
```

Comment:
I wasn’t expecting this to be the fix, nor was it the outcome I aimed for, but it does solve the issue.